packages/client3/components/design-system/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,87 @@
+import clsx from "clsx";
+import { useRouter } from "next/router";
+
+import { FreeArtistsLogoFull } from "../logo";
+import { Toggle, ToggleProps } from "../toggle";
+import { Heading } from "../typography";
+
+interface SideBarProps extends ToggleProps {
+  /**
+   * Sidebar Labels
+   */
+  labels: Array<string>;
+  className?: string;
+  pathName?: string;
+  storyBookMode?: string;
+}
+
+export function SideBar({
+  labels,
+  className,
+  states,
+  pathName,
+  storyBookMode,
+}: SideBarProps) {
+  const router = useRouter();
+  const handleClick = (event: any, href: string, pathName?: string) => {
+    if (pathName) {
```

Comment:
I believe this was added to show the component in the storybook as per our discussion. I kind of feel this is not the right approach as the component shouldn't be concerned about the storybook. I'm not sure about the right way to handle this though, maybe mocking the useRouter could help.